### PR TITLE
MA-11: Create partners page

### PR DESCRIPTION
**Pull request by vovaklh**

### CODE REVIEW

**Comment by vovaklh** [comment]
https://user-images.githubusercontent.com/61093713/197740996-168c2080-2a44-4bf9-a8ab-d9af5f2f8aeb.mp4

File: lib/presentation/pages/partners_page/sections/become_partner_section.dart [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _CareSericeButton extends StatelessWidget {
```
Comment (vitalii-petrun) [inline lib/presentation/pages/partners_page/sections/become_partner_section.dart:41]: maybe CareService ?

Comment (vitalii-petrun) [inline lib/presentation/pages/partners_page/sections/become_partner_section.dart:41]: maybe CareService ?

File: lib/presentation/pages/partners_page/sections/become_partner_section.dart [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return Hover(
+      builder: (_) {
+        return Container(
+          decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(8.0),
+            border: Border.all(color: context.color.joinButtonBorder),
+            color: context.color.careSeviceButton,
```
Comment (vitalii-petrun) [inline lib/presentation/pages/partners_page/sections/become_partner_section.dart:57]: service ?

File: lib/presentation/pages/partners_page/sections/table_section.dart [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+class _ReferralBonusRow extends StatelessWidget {
+  final int referralBonus;
+  final int cachback;
+  final int numberOfLevels;
+  final String bonus;
+
+  const _ReferralBonusRow({
+    Key? key,
+    required this.referralBonus,
```
Comment (vitalii-petrun) [inline lib/presentation/pages/partners_page/sections/table_section.dart:160]: put key in the end of constructor

File: lib/presentation/pages/partners_page/sections/table_section.dart [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+    required this.cachback,
+    required this.numberOfLevels,
+    required this.bonus,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: context.color.referralBonusCell,
+      padding: const EdgeInsets.only(top: 5, bottom: 20, left: 20, right: 20),
```
Comment (vitalii-petrun) [inline lib/presentation/pages/partners_page/sections/table_section.dart:170]: use even numbers in UI building

File: lib/presentation/themes/src/theme_colors.dart [hunk 5]
```diff
@@ -13,6 +13,12 @@ class ThemeColors extends ThemeExtension<ThemeColors> {
   final Color footerBackground;
   final Color localeDropdownBackground;
   final Color localeDropdownOpacity;
+  final Color incomeSectionBackground;
+  final Color joinButtonBorder;
+  final Color partnersCardBackground;
+  final Color partnersCardContentBackground;
+  final Color careSeviceButton;
```
Comment (vitalii-petrun) [inline lib/presentation/themes/src/theme_colors.dart:20]: careService

File: lib/presentation/widgets/header/components/expanded_menu.dart [hunk 6]
```diff
@@ -3,6 +3,12 @@ part of '../header.dart';
 class _ExpandedMenu extends StatelessWidget {
   const _ExpandedMenu({Key? key}) : super(key: key);
 
+  void _goToPartnersPage(BuildContext context) {
+    Navigator.of(context).push(MaterialPageRoute(
+      builder: (_) => PartnersPage(),
+    ));
+  }
```
Comment (vitalii-petrun) [inline lib/presentation/widgets/header/components/expanded_menu.dart:10]: do we need comma there ?

**Review by vitalii-petrun — CHANGES_REQUESTED** [review]
overall looks very nice, but there's little remarks

File: lib/presentation/widgets/header/components/expanded_menu.dart (hunk 6 above)
Comment (vovaklh) [reply lib/presentation/widgets/header/components/expanded_menu.dart:10]: Yes

**Review by vitalii-petrun — DISMISSED** [review]
good

**Review by vitalii-petrun — APPROVED** [review]

**Pull request merged**